### PR TITLE
[aon_timer,dv] Testplan Update for AON Timer

### PR DESCRIPTION
**Pull request by ctopal**
This commit includes updates on the testplan of AON Timer.

Signed-off-by: Canberk Topal <ctopal@lowrisc.org>

### CODE REVIEW

File: hw/ip/aon_timer/data/aon_timer_testplan.hjson [hunk 1]
```diff
@@ -12,22 +12,60 @@
...
+            configuration registers of watchdog timer.
+
+            **Stimulus**:
+            - Register writes to WDOG_BARK_THOLD, WDOG_BITE_THOLD, WKUP_THOLD, WKUP_CTRL
+
+            **Checks**:
+            - If we are changing WDOG_BARK_THOLD to be lower than the current WDOG_COUNT
+            watchdog timer should raise the interrupt `wdog_timer_bark` .
+            - If we are changing WDOG_BITE_THOLD to be lower than the current WDOG_COUNT
+            watchdog timer should trigger a reset request .
```
Comment (cindychip) [inline hw/ip/aon_timer/data/aon_timer_testplan.hjson:45]: super nit: looks like line 43 and 45 has an extra space before `.` :)

File: hw/ip/aon_timer/data/aon_timer_testplan.hjson [hunk 2]
```diff
@@ -12,22 +12,60 @@
...
+            - If we are changing WKUP_THOLD to be lower than the current WKUP_COUNT
+            wakeup timer should raise the interrupt `wkup_timer_expired`.
+            - WKUP_COUNT should reflect expected passing of real time regardless of
+            prescaler value.
+            '''
+      milestone: V1
```
Comment (cindychip) [inline hw/ip/aon_timer/data/aon_timer_testplan.hjson:51]: Just double check: Is this intended to be a V1 test? Same for wdog_lock test.

**Review by cindychip — COMMENTED** [review]
Thanks for the PR :)
Looks great with a few questions. 
Also one more general questions: are these all the tests you plan to implement for aon_timer? I also saw `wake_cause` register, do you plan to check that value in some test?

**Comment by ctopal** [comment]
Thanks for the review! I fixed the spacing problem and changed the milestone for stress test to be V2. For wdog_lock test, I think it's reasonably straightforward to check so I think might be better to put it into V1.

File: hw/ip/aon_timer/data/aon_timer_testplan.hjson [hunk 3]
```diff
@@ -12,22 +12,60 @@
...
+            the programmers guide.
 
             **Stimulus**:
-            - TBD
+            - Register writes to Watchdog/Wakeup COUNT, CTRL and THOLD registers
 
             **Checks**:
-            - TBD
+            - If we are changing WDOG_BARK_THOLD to be lower than the current WDOG_COUNT
+            watchdog timer should raise the interrupt `wdog_timer_bark` .
```
Comment (cindychip) [inline hw/ip/aon_timer/data/aon_timer_testplan.hjson:23]: maybe I did not comment the right place, still see some extra space before the `.`
Same for line 25.

**Review by cindychip — APPROVED** [review]
LGTM with one more small nit.

**Pull request merged**